lib/util/init.config.js

```diff
@@ -239,13 +251,49 @@ const Init = {
             propertiersCur[fieldName] = defaultPropsCur[fieldName];
         }
     },
+    /**
+     * returns list of files that need to be updated
+     * @param {string} projectVersion version found in config file of the current project
+     * @returns {string[]} relevant files with path that need to be updated
+     */
+    _getForcedUpdateList(projectVersion) {
+        // list of files that absolutely need to get overwritten, no questions asked, when upgrading from a version lower than the given.
+        // !newest entry on top, sorted by version number!
+        let forceIdeConfigUpdate;
+        if (File.existsSync(Util.configFileName)) {
+            try {
+                forceIdeConfigUpdate = File.readJsonSync(
+                    path.resolve(__dirname, Util.boilerplateDirectory, 'forcedUpdates.json')
+                );
+            } catch (ex) {
+                Util.logger.error(`${ex.code}: ${ex.message}`);
+                return [];
+            }
+        }
+        // return all if no project version was found or only changes from "newer" versions otherwise
+        const relevantForcedUpdates = [];
+        for (let i = 0; i < forceIdeConfigUpdate.length; i++) {
```

Comment:
any reason for not using for of?

Reply:
none, went old school here but upgraded it now. thx for the reminder :)